pkg/ovsdb/dial_windows_test.go

```diff
@@ -0,0 +1,24 @@
+package ovsdb
+
+import (
+	"testing"
+
+	"github.com/Microsoft/go-winio"
+)
+
+func TestPipeDialNet(t *testing.T) {
+	network := "pipe"
+	address := `\\.\pipe\ovsdbclienttestpipe`
+	l, err := winio.ListenPipe(address, nil)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Why Fatal not Error as in other test cases in the main test file?

Reply:
Done. Use Error instead of Fatal.